lte/gateway/Vagrantfile

```diff
@@ -42,8 +42,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     magma.vm.provider "virtualbox" do |vb|
       vb.name = "magma-dev"
       vb.linked_clone = true
-      vb.customize ["modifyvm", :id, "--memory", "8192"]
-      vb.customize ["modifyvm", :id, "--cpus", "4"]
+      vb.customize ["modifyvm", :id, "--memory", ENV.fetch("MAGMA_DEV_MEMORY_MB", "8192")]
```

Comment:
This file is not only used in CI, people also use it locally by doing vagrant up, if we do so local setup will be broken (missing env bar), I guess we can hardcore the new config

Reply:
I think the second parameter of `fetch` is a fallback so it would be the same as before for users without the env variable?

Reply:
Yes, exactly - you can use the variable to override the default value, but if you don't set it, nothing changes.

Reply:
That's dope